src/cider/nrepl.clj

```diff
@@ -632,6 +632,20 @@ if applicable, and re-render the updated value."
              "refresh-clear"
              {:doc "Clears the state of the refresh middleware. This can help recover from a failed load or a circular dependency error."}}})
 
+(def-wrapper wrap-refresh cider.nrepl.middleware.reload/handle-reload
+  {:doc "Reload middleware."
+   :requires #{"clone" #'wrap-print}
+   :handles {"reload"
```

Comment:
```suggestion
   :handles {"cider.clj-reload/reload"
```

New ops should use ns-qualified naming.

(`cider/reload` would not be not sufficiently clear - we should make it clear that this is a wrapper for a specific library)